app/controllers/vendors_controller.rb

```diff
@@ -9,10 +10,21 @@ def show
     @review = Review.new
   end
 
-  def new
-    @vendor = Vendor.new
+  def create
+    @vendor = Vendor.create(vendor_params)
+    @vendor.viewable = false
+    if @vendor.save
+      flash[:notice] = "Vendor added. Pending review."
+      redirect_to vendors_path
+    else
+      flash[:notice] = "Invalid entry"
+      redirect_to :back
+    end
   end
 
-  def create
+  protected
+  def vendor_params
+    params.require(:vendor).permit(:vendor_name, :viewable)
```

Comment:
It looks like you're allowing `viewable` to be passed in via params, but then you're setting `viewable` explicitly to false in the create action. One of these two things should probably be changed! Are you ever going to need to take in `viewable` via your form? If not, remove it from `vendor_params`!